js-envs/src/main/scala/org/scalajs/jsenv/nodejs/JSDOMNodeJSEnv.scala

```diff
@@ -18,6 +18,7 @@ import org.scalajs.jsenv._
 import org.scalajs.core.ir.Utils.escapeJS
 
 class JSDOMNodeJSEnv(
+  jsdomDirectory: File,
```

Comment:
Consequently, this could be generalized to an arbitrary sequence of directories, and not be tied to `jsdom` installation per se. 

Or without changing the API, simply use `addArgs` to pass command-line arguments.